package exception;

public class Example1 {

	public static void main(String[] args) {
		
		System.out.println("main method start");
		

		try
		{
			int k = 9/0;
			System.out.println(k);
		}
		catch(ArithmeticException e)
		{
			e.printStackTrace();
		}
		
		System.out.println("main method end");
	}
}
